int16 pin
int16 pozicija
time cas
